resources/views/achievement/edit.blade.php

```diff
@@ -0,0 +1,216 @@
+
+                        <hr>
+
+                        <table class="table">
+                            <thead>
+                            <tr>
+                                <th>User count</th>
+                                <th>Achieved</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td>Members only</td>
+                                <td>{{ count($achievement->current(true)) }}</td>
+                            </tr>
+                            <tr>
+                                <td>All users</td>
+                                <td>{{ count($achievement->current(false)) }}</td>
+                            </tr>
+                            </tbody>
+                        </table>
+
+                    @endif
+                </div>
+
+            </div>
+
+            <div class="panel-footer clearfix">
+
+                <button type="submit" class="btn btn-success pull-right" style="margin-left: 15px;">Submit</button>
+
+                <a href="{{ route("achievement::list") }}" class="btn btn-default pull-right">Cancel</a>
+
+            </div>
+
+        </div>
+
+    </form>
+
+    @if(!$new)
+
+        <form method="post" action="{{ route("achievement::image", ["id" => $achievement->id]) }}"
+              enctype="multipart/form-data">
+
+            {!! csrf_field() !!}
+
+            <div class="panel panel-default">
+
+                <div class="panel-heading">
+                    Update icon
+                </div>
+
+                <div class="panel-body">
+
+                    @if($achievement->image)
+
+                        <img src="{!! $achievement->image->generateImagePath(700,null) !!}" width="100%;">
+
+                    @else
+                        <p>
+                            &nbsp;
+                        </p>
+                        <p style="text-align: center;">
+                            This achievement has no icon yet. Upload one now!
+                        </p>
+                    @endif
+
+                    <hr>
+
+                    <div class="form-horizontal">
+
+                        <div class="form-group">
+                            <label for="image" class="col-sm-4 control-label">New achievement icon</label>
+                            <div class="col-sm-8">
+                                <input class="form-control" id="image" type="file" name="image">
+                            </div>
+                        </div>
+
+                    </div>
+
+                </div>
+
+                <div class="panel-footer clearfix">
+                    <button type="submit" class="btn btn-success pull-right">
+                        Replace icon
+                    </button>
+                </div>
+
+            </div>
+
+        </form>
+
+    @endif
+
+    </div>
+
+@endsection
+
+@section('stylesheet')
```

Comment:
Deze CSS moet in een LESS file, zie de pinned message van Bouke in de commissieslack.